src/user_management.py

```diff
@@ -0,0 +1,56 @@
+from faker import Faker
+import json
+import re
+from datetime import datetime
+
+fake = Faker()
+
+
+def generate_fake_user():
+
+    try:
+        return {
+            "name": fake.first_name(),
+            "birthday": fake.date_of_birth().strftime("%d.%m.%Y"),
+            "phone": re.sub(r"\D", "", fake.phone_number())[:10],
+            "email": fake.email(),
+            "address": fake.address(),
+        }
+    except Exception as e:
+        print(f"Error generating fake user: {e}")
+        return []
+
+
+def generate_fake_users(count=25):
+    return [generate_fake_user() for _ in range(count)]
+
+
+def load_users() -> list:
+
+    users_data = generate_fake_users()
+
+    return users_data
+
+
+def save_users(users_data, filename="users_data.json"):
+    try:
+        with open(filename, "w") as file:
+            json.dump(users_data, file, indent=4)
+        print("Users data saved successfully.")
+    except Exception as e:
+        print(f"Error saving users data: {e}")
+
+
+def load_or_generate_users():
+    try:
+        with open("../../src/users_data.json", "r") as file:
```

Comment:
add this file to .gitignore:
users_data.json
.vscode/ (this is as additional pls)